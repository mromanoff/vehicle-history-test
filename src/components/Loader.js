import React from 'react';
import { BarLoader} from 'react-css-loaders';

const Loader = () => <BarLoader size={7} color="#FF8C00"/>;

export default Loader;
