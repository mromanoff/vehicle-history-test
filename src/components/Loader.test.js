import React from 'react';
import Loader from './Loader';

it('Loader renders correctly', () => {
  const wrapper = shallow(<Loader />);
  expect(wrapper).toMatchSnapshot();
});
