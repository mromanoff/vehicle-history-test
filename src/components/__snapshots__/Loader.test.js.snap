// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Loader renders correctly 1`] = `
<div
  className="loader-wrapper"
>
  <div
    className="loader"
  >
    <div
      className="loader-inner"
    />
  </div>
</div>
`;
